test/libsolidity/syntaxTests/visibility/interface/function_default.sol

```diff
@@ -1,6 +1,3 @@
 interface I {
-	function f();
+	function f() external;
```

Comment:
`default` in the name suggests that we do not want to have visibility here :)

Reply:
Makes a lot of sense :)